### ISSUE TITLE
File problem creating directory

### ISSUE THREAD

**Issue by marcullo** [marcullo/image-renderer#1]
###### System: Windows 10 WSL2

There is a problem with compilation on the WSL2. I don't know the command used for the build. It seems it was just

```
cmake .
```

which shouldn't be used (reproduced the same log as below on my Ubuntu machine).

<details><summary>log</summary>

```
-- Output directory: /home/crash/projects/peach/zadania/strawberry/image-renderer
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at samples/CMakeLists.txt:8 (file):
  file problem creating directory: /renders


-- Renders in: /renders
CMake Error at samples/CMakeLists.txt:15 (file):
  file problem creating directory: /renders/basic


CMake Error at samples/CMakeLists.txt:15 (file):
  file problem creating directory: /renders/crop_extend


CMake Error at samples/CMakeLists.txt:15 (file):
  file problem creating directory: /renders/squared


CMake Error at /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/ExternalProject.cmake:1711 (file):
  file problem creating directory: /external/bitmap
Call Stack (most recent call first):
  /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/ExternalProject.cmake:3223 (_ep_set_directories)
  CMakeLists.txt:13 (ExternalProject_Add)


CMake Error at /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/ExternalProject.cmake:1713 (message):
  dir '/external/bitmap' does not exist after file(MAKE_DIRECTORY)
Call Stack (most recent call first):
  /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/ExternalProject.cmake:3223 (_ep_set_directories)
  CMakeLists.txt:13 (ExternalProject_Add)


-- Configuring incomplete, errors occurred!
See also "/home/crash/projects/peach/zadania/strawberry/image-renderer/sdk/graphics/bitmap-subbuild/CMakeFiles/CMakeOutput.log".

CMake Error at /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/FetchContent.cmake:900 (message):
  CMake step for bitmap failed: 1
Call Stack (most recent call first):
  /home/crash/cmake-3.17.3-Linux-x86_64/share/cmake-3.17/Modules/FetchContent.cmake:936 (__FetchContent_directPopulate)
  cmake/fetch_raw_lib.cmake:8 (FetchContent_Populate)
  sdk/graphics/CMakeLists.txt:6 (fetch_raw_lib)


-- Configuring incomplete, errors occurred!
See also "/home/crash/projects/peach/zadania/strawberry/image-renderer/CMakeFiles/CMakeOutput.log".
```

</details>

**Comment by marcullo** [comment]
##### Workaround for not using `make`

Assuming you are in the project root folder,

1. Configure the project. Select one (debug is more verbose for the user, so I'd select it).

(debug)
```
export PROJECT_PATH=$PWD; cmake -S $PROJECT_PATH  -B $PROJECT_PATH/_build/cmake -DCMAKE_VERBOSE_MAKEFILE=TRUE  -GNinja -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=$PROJECT_PATH/_build -DCMAKE_SIZE=size -DCMAKE_BUILD_TYPE=Debug
```

(release)
```
export PROJECT_PATH=$PWD; cmake -S $PROJECT_PATH  -B $PROJECT_PATH/_build/cmake -DCMAKE_VERBOSE_MAKEFILE=TRUE  -GNinja -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=$PROJECT_PATH/_build -DCMAKE_SIZE=size -DCMAKE_BUILD_TYPE=Release
```

2. Build the project:
```
cmake --build $PROJECT_PATH/_build/cmake --target all
```

3. Clean the project if needed:
```
rm -rf $PROJECT_PATH/_build
```

**Comment by marcullo** [comment]
Fixed by c5e08339d3fb1b138c76904869ba6df514134e7c
